topiary-tree-sitter-facade/src/query_match.rs

```diff
@@ -38,8 +39,8 @@ mod wasm {
 
     impl<'tree> QueryMatch<'tree> {
         #[inline]
-        pub fn pattern_index(&self) -> u32 {
-            self.inner.pattern()
+        pub fn pattern_index(&self) -> usize {
+            self.inner.pattern().try_into().unwrap()
```

Comment:
This `unwrap()` could do with a comment. I suppose it's not going to panic for any platform where word size is at least 32 bits, so most of what we care about for Topiary. It's still good to have it written down, I guess

Reply:
Done in 5c35e051371bda5228